libraries/AP_HAL/CANIface.h

```diff
@@ -32,31 +33,30 @@ struct AP_HAL::CANFrame {
     static const uint32_t FlagRTR = 1U << 30;                  ///< Remote transmission request
     static const uint32_t FlagERR = 1U << 29;                  ///< Error frame
 
+#if HAL_CANFD_SUPPORTED
```

Comment:
we should move the init of these constants to the CANIFace.cpp so the header is the same on systems with/without CANFD

Reply:
can't its being used inside the class definitions itself.

Reply:
it seems like the FD and non FD firmwares are so far apart that it doesn't even seem to change anything, tested it by keeping constant between CubeOrange and CubeBlack.